analysis.ml

```diff
@@ -101,6 +102,16 @@ module VariableMap = struct
       | Some a, Some b -> Some (InstrSet.union a b) in
     merge merge_one
 
+  let inter a b =
```

Comment:
This could be given a much shorter (and probably more efficient) definition using [`Map.merge`](http://caml.inria.fr/pub/docs/manual-ocaml/libref/Map.S.html#VALmerge), but `merge` above shadows the name `merge`, so we should rename into `union` I guess.